build-tools/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -114,7 +114,8 @@ public class ElasticsearchNode implements TestClusterConfiguration {
         "path.repo",
         "discovery.seed_providers",
         "cluster.deprecation_indexing.enabled",
-        "cluster.initial_master_nodes"
+        "cluster.initial_master_nodes",
+        "xpack.security.enabled"
```

Comment:
This has the effect that allows you to enable security via the integ_test distribution (in x-pack or core). I think that is a good thing especially if we want to start using the integ_test distro for all module/plugin REST tests.

Reply:
It seems that security is disabled by default for `INTEG_TEST` distribution. 
https://github.com/elastic/elasticsearch/blob/0ac81ce52a5474082465d48d68a2a8d5363e758f/build-tools/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchCluster.java#L355-L356

I think that means most FIPS related stuff will *not* be excercised in tests with `INTEG_TEST` distribution even the `fips.gradle` file configures a bunch of settings, e.g. `xpack.security.fips_mode.enabled`. Theoretically, this is probably a reduced coverage for FIPS since we are currently forcing `DEFAULT` distribution for FIPS tests? 

That said, there are still tons of tests that use the `DEFAULT` distribution so that coverage is not really an issue. And we can later look into whether it is necessary and how to enable security for INTEG_TEST.

Reply:
> Theoretically, this is probably a reduced coverage for FIPS since we are currently forcing DEFAULT distribution for FIPS tests?

I don't think this will reduce coverage since security is also disabled for FIPS tests with the DEFAULT distribution. Individual tests can enable security. Before this change they could only enable security if they used the default distribution, but with this change they can override for any REST test.  https://github.com/elastic/elasticsearch/pull/91101/files#diff-93a50105f87a94f0142b8885f2cadb8d4817597a8a8410310f40fe6180643821R72

Reply:
I missed that. Thanks!